LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 Vinted
```

Comment:
I think it should be full name "Vinted, UAB"

Reply:
@laurynas updated